module/solutions/helper.js

```diff
@@ -2297,3 +2397,4 @@ function _generateLink(appsPortalBaseUrl, prefix, solutionLink, solutionType) {
 
   return link;
 }
+const usersHelper = require(MODULES_BASE_PATH + "/users/helper");
```

Comment:
@VISHNUDAS-tunerlabs 2 ways to fix this one is to create a new helper file calling userHelperV2 or what @ankitpws suggested.